tests/protractor/priority/priority.js

```diff
@@ -26,24 +23,21 @@ app.controller('PriorityCtrl', function Chat($scope, $firebase) {
     if ($scope.message !== '') {
       // Add a new message to the messages list
       var priority = $scope.messages.length;
-      $scope.messages.$inst().$push({
+      $scope.messages.$add({
```

Comment:
This wasn't your original intent of this bit of code. It looks like you wanted to exercise adding records from the server side and not client. But it seems like a good exercise of the code base and provides a functional test. I think that adding from the server is sufficiently exercised in the test unit that ensures existing data survives a page refresh.